src/utils/listingUtils.js

```diff
@@ -1,44 +1,66 @@
-export const NEEDS_SHEET_NAME = "Form Responses 1"
+export const NEEDS_SHEET_NAME = "Public needs responses"
 
 export const NEEDS_SHEET_COLUMN_INDICES = {
   createdAt: 0,
+  onBehalfOf: 1,
   name: 2,
   pronouns: 3,
   location: 4,
   preferredContactMethod: 5,
   contact: 6,
+  interestedInConnecting: 7,
   neighborhoodName: 8,
-  public: 9,
-  isFirstNeed: 11,
 
   // FINANCIAL NEED
-  isFinancialNeed: 12,
-  financial_needFrequency: 13,
-  financial_needTiming: 14,
-  financial_minFundingNeed: 17,
-  financial_maxFundingNeed: 18,
-  financial_fundingMethod: 19,
+  isFinancialNeed: 9,
+  financial_needFrequency: 10,
+  financial_needTiming: 11,
+  financial_minFundingNeed: 14,
+  financial_maxFundingNeed: 15,
+  financial_fundingMethod: 16,
 
   // SUPPIES/ERRANDS NEED
-  isSuppliesNeed: 20,
-  supplies_needFrequency: 21,
-  supplies_needTiming: 22,
-  supplies_details: 23,
-  supplies_neighborhood: 24,
-  supplies_store: 25,
-  supplies_shoppingList: 26,
+  isSuppliesNeed: 18,
+  supplies_needFrequency: 19,
+  supplies_needTiming: 21,
```

Comment:
There is also now one new question after timing, which is about covering cosots, so maybe we can call `supplies_costCoverage`

Reply:
I missed that one! I'll update now.

Reply:
@mewtaylor i'm a bit confused. Can you review the column headers for supplies on https://docs.google.com/spreadsheets/d/1TlDMhnDDuOAUzBTlW05t5mtoKUKcKp7Fmg8VL4PJOCU/edit#gid=2052572100 not sure which one is for the cost coverage.

Reply:
Yes! it's the one that says "I would", V.

Reply:
@mewtaylor should be fixed now